project/BuildSettings.scala

```diff
@@ -95,4 +101,17 @@ object BuildSettings {
         oldStrategy(x)
     }
   )
+
+  lazy val scoverageSettings = Seq(
```

Comment:
Doesn't seem we submit data to coveralls?

Reply:
Indeed, I was planning to ask you if you made up your mind between `codecov` and `coveralls`.

Could you add `COVERALLS_REPO_TOKEN` to this repo please ? I will add it.